### PR TITLE
Refactor Suspense component

### PR DESCRIPTION
**Pull request by lucasnestrojil**
## Summary
- Refactor Suspense component 
- Deleted maxDuration prop

## Jira Card
https://trello.com/c/yugeKn7T/229-actualizar-componentes-customsuspense

### CODE REVIEW

File: generators/app/templates/src/app/components/Suspense/index.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React, { Suspense } from 'react';
+
+import Loading from '../Spinner/components/loading';
+
+interface Props {
+  fallback?: React.ElementType;
+  children: React.ReactNode;
+}
+
+function CustomSuspense({ fallback = Loading, children }: Props) {
```
Comment (carlosbollero) [inline generators/app/templates/src/app/components/Suspense/index.tsx:10]: If fallback is optional then the default value isn't needed. If it is, then remove the optional marker.

**Review by carlosbollero — CHANGES_REQUESTED** [review]

File: generators/app/templates/src/app/components/Suspense/index.tsx (hunk 1 above)
Comment (damfinkel) [reply generators/app/templates/src/app/components/Suspense/index.tsx:10]: This doesn't work in TS I think. If you define it as not optional, the user must pass something to match the interface. If that's the case, I'd leave it as it is

Comment (damfinkel) [reply generators/app/templates/src/app/components/Suspense/index.tsx:10]: @pabloferro can you confirm this?

**Review by damfinkel — APPROVED** [review]

File: generators/app/templates/src/app/components/Suspense/index.tsx (hunk 1 above)
Comment (pabloferro) [reply generators/app/templates/src/app/components/Suspense/index.tsx:10]: Capi is right.
We should provide default values to optional props or handle the undefined case.

**Review by carlosbollero — APPROVED** [review]

**Pull request merged**